components/brave_wallet_ui/components/shared/create-placeholder-icon/index.tsx

```diff
@@ -118,7 +116,11 @@ export function withPlaceholderIcon<
 
     const remoteImage = React.useMemo(() => {
       if (isRemoteURL) {
-        return isStorybook ? tokenImageURL || '' : `chrome://image?${tokenImageURL}`
+        return isStorybook
```

Comment:
If you use `createSanitizedImageUrl` from `components/common/create_sanitized_image_url.ts` to create this Url then it will also work in storybook (same for all other uses of chrome://image)

Reply:
Removed this `Storybook` check, other checks for `Storybook` are instance's where we use `chrome-untrusted-iframes` which seem to break in `Storybook`.